### DISCUSSION TITLE
Running a workflow when a specific file is modified

### DISCUSSION THREAD

**Discussion by transcental** [community/community#25750] [Actions]
<p>Hi,</p>
<p>I have a python package on my repo, and I would like it to automatically deploy to PyPi when I modify the <code>setup.cfg</code> file.</p>
<p>Is this possible?<br>
I’ve got a workflow to deploy to PyPi, but I don’t know how to make it run only when I modify a specific file.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It depends a bit on the event you want to trigger the workflow, but with a <code>push</code> trigger you could <a href="https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths">use a <code>paths</code> filter</a> that selects <code>setup.cfg</code>, or any other file you want.</p>
<pre><code class="lang-yaml">on:
  push:
    paths:
      - 'setup.cfg'
</code></pre>

**Comment by transcental** [comment]
<p>Thank you! That should work perfectly!</p>
